### PR TITLE
README: Update corectness for examples

### PR DESCRIPTION
**Pull request by jeremylt**
This updates the top level README. I'm not a big fan of this duplication and am open to removing/adjusting it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,39 +141,45 @@ To build the examples, set the `MFEM_DIR`, `PETSC_DIR` and `NEK5K_DIR` variables
 and run:
 
 ```console
-# libCEED examples on CPU and GPU
 cd examples/ceed
```
Comment (jedbrown) [inline README.md]: Either the above is correct or the `cd ceed` below, but not both.

File: README.md [hunk 2]
```diff
@@ -141,39 +141,45 @@ To build the examples, set the `MFEM_DIR`, `PETSC_DIR` and `NEK5K_DIR` variables
...
+
+# libCEED examples on CPU and GPU
+cd ceed
 make
 ./ex1 -ceed /cpu/self
 ./ex1 -ceed /gpu/occa
-cd ../..
+cd ../...
```
Comment (jedbrown) [inline README.md]: Can't be right.

File: README.md [hunk 3]
```diff
@@ -141,39 +141,45 @@ To build the examples, set the `MFEM_DIR`, `PETSC_DIR` and `NEK5K_DIR` variables
...
-./bps -problem bp1 -ceed /gpu/occa
+./bps -problem bp2 -ceed /gpu/occa
+./bps -problem bp3 -ceed /cpu/self
+./bps -problem bp4 -ceed /gpu/occa
+./bps -problem bp5 -ceed /cpu/self
+./bps -problem bp6 -ceed /gpu/occa
 cd ../..
 
-cd navier-stokes
+examples/cd navier-stokes
```
Comment (jedbrown) [inline README.md]: typo

**Comment by jeremylt** [comment]
Fixed. I should have seen those.

**Pull request merged**